Get authenticated user

christopher-dG opened the pull request:
It seems a bit weird that this package uses the term `Owner` instead of `User`, but oh well.

DilumAluthge commented:
Presumably that's because an owner can be a user or an organization?

christopher-dG commented:
True, it's just not good for my tab-completion-powered API exploration 😅 

Windows nightly is failing, due to some unrelated issue. Should probably allow failure on those.

DilumAluthge commented:
Seems fine to allow failures on Windows nightly.

fredrikekre commented on src/owners/owners.jl:
```diff
@@ -44,6 +44,11 @@ isorg(owner::Owner) = something(owner.typ, "") == "Organization"
 
 @api_default owner(api::GitHubAPI, owner_obj::Owner; options...) = owner(api, name(owner_obj), isorg(owner_obj); options...)
 
+@api_default function owner(api::GitHubAPI; options...)
```
`whoami` would be a nice name for this method.

christopher-dG commented:
Renamed to `whoami`. The README changes are big but I just aligned the tables automatically with my editor. The only line I added is the `whoami` line [here](https://github.com/JuliaWeb/GitHub.jl/blob/56c9aa23c593d76ae345deb8377ef7158cfd5798/README.md#users-and-organizations).

DilumAluthge commented on .travis.yml:
```diff
@@ -9,12 +9,16 @@ julia:
 
+jobs:
+    allow_failures:
+        - julia: nightly
```
```suggestion
        - julia: nightly
          os: windows
```

christopher-dG replied:
I'm usually of the opinion that nightly can fail on any OS it wants, but this is fine too.

DilumAluthge reviewed (changes requested):
I would suggest only allowing failures on Julia nightly on Windows.

DilumAluthge reviewed (approved):
LGTM.
